components/session-based-conditional-auth-functions/src/test/resources/testng.xml

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Identity-connector-session-validation-test-suite">
+    <test name="session-validation-authenticator-all" parallel="false">
+        <classes>
+            <class name="org.wso2.carbon.identity.extensions.conditional.auth.functions.test.SessionModelTest"/>
+            <class name="org.wso2.carbon.identity.extensions.conditional.auth.functions.test.ConditionalAuthFunctionTest"/>
+            <class name="org.wso2.carbon.identity.extensions.conditional.auth.functions.test.SessionValidationUtilTest"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
Always we need to add an extra line at the end of file

Reply:
newline added for all missing places